test/typescript-tenant.ts

```diff
@@ -59,6 +59,19 @@ test('Fetch', async() => {
     expect(collection.User.length).toBe(userData.length)
 })
 
+
+test('Group by with params', async() => {
```

Comment:
I added this test here because I did not know where to add it, please let me know if I should move it.